docs/content/en/docs/resources/_index.md

```diff
@@ -28,7 +28,17 @@ description: "Additional resources to learn about Tetragon"
 
 [Detecting and Blocking log4shell with Isovalent Cilium Enterprise](https://isovalent.com/blog/post/2021-12-log4shell) - Jed Salazar, 2021
 
+[Strengthen your system with Tetragon's eBPF-based Security Observability and Runtime Enforcement capabilities](https://b-nova.com/en/home/content/strengthen-your-system-with-tetragons-ebpf-based-security-observability-and-runtime-enforcement-capabilities) - Stefan Welsch, 2022
+
+[Can I Use Tetragon without Cilium? Yes!](https://isovalent.com/blog/post/can-i-use-tetragon-without-cilium-yes/) - [Dean Lewis](https://twitter.com/saintdle), 2023
```

Comment:
Sorry just a nit, can you put in the order of more recent to the least recent, as we do with conference talks just above? :)

Reply:
with the two above from 2021